indico/modules/events/blueprint.py

```diff
@@ -58,6 +59,7 @@
 
 # Misc
 _bp.add_url_rule('/event/<int:event_id>/key-access', 'key_access', RHEventAccessKey, methods=('POST',))
+_bp.add_url_rule('/event/<int:event_id>/api/info-for-series', 'single_event_api', RHSingleEventAPI, methods=('GET',))
```

Comment:
```suggestion
_bp.add_url_rule('/event/<int:event_id>/api/info-for-series', 'single_event_api', RHSingleEventAPI)
```

GET is already the default ;)